hawkular-rest-status/src/main/java/org/hawkular/commons/rest/status/RestStatusHandler.java

```diff
@@ -35,33 +34,42 @@
 import com.wordnik.swagger.annotations.Api;
 
 /**
- * REST endpoint for status.
+ * REST endpoint for baseStatus.
  *
- * <p>This handles the {@code /status} URL and outputs information read from the META-INF/MANIFEST.MF file together
- * with whatever information injected into it by maps of strings qualified by {@link RestStatusInfo @RestStatusInfo}.
- * There can be any number of such maps (including none) and all of them will be included in the output.
+ * <p>
+ * This handles the {@code /baseStatus} URL and outputs information read from the META-INF/MANIFEST.MF file of the
+ * present servlet context together with whatever information injected into it by maps of strings qualified by
+ * {@link RestStatusInfo @RestStatusInfo}. There can be any number of such maps (including none) and all of them will be
+ * included in the output.
  *
  * @author Jay Shaughnessy
  * @author Lucas Ponce
  * @author Lukas Krejci
  */
-@Path("/status")
-@Api(value = "/status", description = "Status of the component service.")
+@Path("/baseStatus")
+@Api(value = "/baseStatus", description = "Status of the component service.")
 @ApplicationScoped
 public class RestStatusHandler {
-    @Inject @RestStatusInfo
+    @Inject
+    @RestStatusInfo
     private Instance<Map<String, String>> details;
 
-    private volatile Map<String, String> status;
+    private final Object baseStatusLock = new Object();
```

Comment:
+1. Synchronizing on `this` is potentially dangerous, so this is better. Thanks.